CMakeLists.txt

```diff
@@ -39,17 +36,50 @@ endforeach ()
 
 include (esma)
 
-ecbuild_declare_project()
+list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/@jedi-cmake")
```

Comment:
Can you remind me what the `@` does in the path?

Reply:
@climbfuji Awww nuts. That's on me. With `mepo` a repo can be cloned as:

* `@jedi-cmake`
* `jedi-cmake@`
* `jedi-cmake`

where the `@` is what we use as a "signal" that you are in a new repository (if desired). For example, GEOSgcm uses a prefix-`@`, some other repos use a suffix-`@` (since that doesn't screw up tab completion), and some use no-`@` (e.g., MAPL). It's sort of up to the fixture maintainer what they want to do.

What I need to do is generalize this for any of those styles. We have a [function in ESMA_cmake](https://github.com/GEOS-ESM/ESMA_cmake/blob/main/esma_support/esma_mepo_style.cmake) that makes this general.

Or maybe, for here, we don't use the `@`? I'll have to consult with @amdasilva as it is his repo.

Reply:
Not sure if I understand better now ;-) either way, fine with me!

Reply:
I should be able to fix that up on Monday. Today is already full with tasks. :)